docs/EN_US/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml

```diff
@@ -530,6 +530,18 @@
               <entry>Upper limit of a CSV line read in MB.</entry>
             </row>
 
+            <row>
+              <entry><emphasis>validateFileType</emphasis></entry>
+
+              <entry>Default true</entry>
+
+              <entry>If false, the engines use the definition in the ECL
+              workunit and ignores the file type in the DFU record. This check
+              is always ignored if the ECL is reading a CSV or a fixed record
+              width flat file. If the ECL is reading XML or JSON, and there is
+              a mismatch, it issues a warning not an error.</entry>
```

Comment:
> This check is always ..

it's not that clear that this is the section is the beginning of a description of what happens if the option is true, I'd change it to : 
If true, this check is always